source/model/clazzFiltering.ts

```diff
@@ -0,0 +1,5 @@
+import { GeneralFiltering } from "./generalFiltering";
+
+export interface ClazzFiltering extends GeneralFiltering {
```

Comment:
follow the names on the server -> its ClazzFilter not ClazzFiltering
also there is no object serverside called GeneralFiltering